images/bin/build

```diff
@@ -29,14 +29,18 @@ done
 
 cd $(dirname "${BASH_SOURCE[0]}")/..
 
-docker pull "${base}"
+docker pull $base
```

Comment:
This should remain quoted, as it's never intended to be more than one argument.

Reply:
➕